packages/react-client/src/DeviceManager.ts

```diff
@@ -164,7 +164,7 @@ const handleNotAllowedError = async (constraints: MediaStreamConstraints): Promi
   return await getMedia({ video: false, audio: false }, { video: PERMISSION_DENIED, audio: PERMISSION_DENIED });
 };
 
-const getError = (result: GetMedia, type: AudioOrVideoType): DeviceError | null => {
+const getError = (result: GetMedia, type: TrackKind): DeviceError | null => {
```

Comment:
Imo `TrackType` sounds better than `TrackKind`

Reply:
maybe update the variable name as well to `kind: TrackKind`

Reply:
Agreed with @czerwiukk. lets rename function parameter name to `kind`.